src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -191,6 +191,7 @@ export interface IAnalyticsTriggerNotify {
 
 export interface IAnalyticsTriggerRedirect {
   redirectedTo: string;
+  query?: string;
```

Comment:
Should the query be a required property?

On the other hand, is it necessary to send or could it have been retrieved from search impressions?

Reply:
ML are saying it is necessary, and it's a relatively simple value to send.
Not required because the older Trigger Component is not sending it (although I am thinking of sending it there as well) 🤔

Reply:
It is also not a "search" we are calling the /plan endpoint to prevent hitting the index